docs/azure/sdk/scaling-aspnetcore-apps.md

```diff
@@ -0,0 +1,352 @@
+
+1. Select **Next: Authentication** to progress to the next step.
+1. Select **System assigned managed identity** and choose **Next: Networking**.
+1. Leave the default networking options selected, and then choose **Review + Create**.
+1. After Azure validates your settings, select **Create**.
+
+The service connector will enable a system-assigned managed identity on the container app. It will also assign a role of **Storage Blob Data Contributor** to the identity so it can perform data operations on the storage containers.
+
+#### Connect the key vault
+
+1. In the Azure portal, navigate to your Container App overview page.
+1. On the left navigation, select **Service connector**
+1. On the Service Connector page, choose **+ Create** to open the **Creation Connection** flyout panel and enter the following values:
+    * **Container**: Select the Container App you created previously.
+    * **Service type**: Choose **Key Vault**.
+    * **Subscription**: Select the subscription you used previously.
+    * **Connection name**: Leave the default value or enter a name of your choosing.
+    * **Key vault**: Select the key vault you created earlier.
+    * **Client type**: Select **.NET**.
+
+        :::image type="content" source="media/scaling-aspnetcore-apps/scaling-connect-keyvault.png" alt-text="A screenshot showing how to create a container app in the Azure Portal.":::
+
+1. Select **Next: Authentication** to progress to the next step.
+1. Select **System assigned managed identity** and choose **Next: Networking**.
+1. Leave the default networking options selected, and then choose **Review + Create**.
+1. After Azure validates your settings, select **Create**.
+
+The service connector will assign a role to the identity so it can perform data operations on the key vault keys.
+
+## 6) Configure and redeploy the app
+
+The necessary Azure resources have been created, so next you'll need to configure your application code to point to those services.
+
+1. Install the following three NuGet packages that are necessary to solve the scaling challenges:
+
+    * **Azure.Identity**: Provides classes to work with the Azure identity and access management services.
+    * **Microsoft.AspNetCore.DataProtection**: Provides services to configure data protection.
+    * **Microsoft.Extensions.Azure**: Provides helpful extension methods to perform core Azure configurations.
+
+    ## [.NET CLI](#tab/cli)
+
+    ```dotnetcli
+    dotnet add package Azure.Identity
+    dotnet add package Azure.Extensions.AspNetCore.DataProtection.Blobs
+    dotnet add package Azure.Extensions.AspNetCore.DataProtection.Keys
+    ```
+
+    ## [Visual Studio](#tab/visual-studio)
+
+    Todo
+
+    ---
+
+1. Update the `Program.cs` file to include the following `using` statements and code:
+
+    ```csharp
+    using Azure.Identity;
+    using Microsoft.AspNetCore.DataProtection;
+    using Microsoft.Extensions.Azure;
+    
+    builder.Services.AddAzureClientsCore();
```

Comment:
What's the motivation for including this line? Does something not work properly if it's excluded? If your goal is to enable forwarding of Azure SDK for .NET logs, use the approach documented here instead: https://docs.microsoft.com/dotnet/azure/sdk/logging#logging-without-client-registration.